packages/mdc-elevation/_mixins.scss

```diff
@@ -25,6 +25,23 @@
 @import "@material/theme/variables";
 @import "./variables";
 
+@mixin mdc-elevation-top-level($query: mdc-feature-all()) {
```

Comment:
For a couple of them I had to do `mdc-<package>-top-level` since `mdc-<package>` was taken. Should I go back and change them all to match for consistency?

Reply:
That's a fair point; it'd probably be less confusing that way (esp. for stuff like mdc-theme).

Also, do you think `mdc-foo-baseline` would work for naming rather than `mdc-foo-top-level`?

Reply:
That would work, though in the case of typography people might think it has something to do with baseline alignment

Reply:
per slcak discussion, going with `mdc-foo-core-styles`